apps/functions-subscription/eslint.config.mjs

```diff
@@ -1,3 +1,13 @@
 import eslintrules from 'eslint-config-rules';
 
-export default [...eslintrules];
+export default [
+  ...eslintrules,
+  {
+    // Disable no-expression-statements only in the
+    // main file containing the endpoint mapping.
+    files: ['src/main.ts'],
+    rules: {
+      'functional/no-expression-statements': 'off',
+    },
+  },
+];
```

Comment:
**thought**: Is it too specific to include here? The `no-expression-statements` is due to the Azure Functions API (`app.http`). What about keeping the file override or move it to workspace level?

Reply:
It could be too specific, but I put it there on purpose.
I think that rule should be disabled in the Azure Function API file, so instead of disable the rule as it was previously (using the `eslint-disable` comment), I created a new entry in the Eslint rules file of the Azure functions.
Furthermore, I wanted to disable the rule only in that file, not somewhere else.

What do you mean with "What about keeping the file override or move it to workspace level?"? IMHO it is already at a workspace level: the rule is listed in the eslint file of the workspace `functions-subscription`.

Reply:
Got it! I understand what you're saying. These kinds of exceptions do seem very specific and are often related to integration with external libraries, this is why I think they might be better handled with `eslint-disable` comment. But let's keep it here and see if it proves to be useful! 🤝